README.md

```diff
@@ -1,2 +1,38 @@
-# release-notes-bot-action
-:horse: Bot to update release notes with new entries
+# Release-notes-bot action
+
+Automatically adds a new release notes entry to once a PR is merged. Two conditions must be met for the action to trigger.
+
+* One of 3 labels must be applied to the PR in order for a release entry to be added:
```

Comment:
This isn't true.

I'm not sure if it matters. I think perhaps just update this to remove.

In the end, if there's release notes, add 'em. That makes the most sense to me.

@EpicEric thoughts?

Reply:
I agree with you. This can be removed.

Reply:
We discussed at sync and if the labels aren't present, we will remove the release notes file and not add. So I need to update all content related to this and make that change.

Reply:
I'm okay with that.